docs/more-info/build-server-setup/tfs-build-vnext.md

```diff
@@ -37,6 +37,12 @@ The VSO build step updates the build number automatically to the GitVersion numb
 GitVersion passes variables in the form of `GitVersion.*` (Eg: `GitVersion.Major`) to TFS Build and also writes `GITVERSION_*` (Eg: `GITVERSION_MAJOR`) environment variables that are available for any subsequent build step. 
 See [Variables](/more-info/variables/) for an overview of available variables.
 
+
+#### Using GitVersion variables in build build name
+To use GitVersion's variables in the build name, just add them in the form `$(GITVERSION_FullSemVer)` into the Build definition's build number string. Then just ensure GitVersion is called with
+`/output buildserver` and it will replace those variables with the calculated version.
```

Comment:
Maybe mentioning that the TFS Build task takes care of that (/output buildserver) might be helpful